src/core/Akka.Cluster/ClusterSettings.cs

```diff
@@ -118,6 +118,7 @@ TimeSpan GetWeaklyUpDuration()
 
             WeaklyUpAfter = GetWeaklyUpDuration();
 
+            UseLegacyHeartbeatMessage = clusterConfig.GetBoolean("use-legacy-heartbeat-message", true);
```

Comment:
needs to default to `false` - we want users to have to use the new message starting with v1.5. It should be opt-out, not opt-in.

Reply:
Got it